test/ArcxAnalyticsSdk.test.ts

```diff
@@ -108,6 +111,35 @@ describe('(unit) ArcxAnalyticsSdk', () => {
         }),
       ).to.be.true
     })
+
+    it('trackTransactions', async () => {
+      const transactionParams = {
+        gas: '0x22719',
+        from: '0xbb4153b55a59cc8bde72550b0cf16781b08ef7b0',
+        to: '0x03cddc9c7fad4b6848d6741b0ef381470bc675cd',
+        data: '0x97b4d89f0...082ec95a',
+      }
+      window.web3 = {
+        currentProvider: sinon.createStubInstance(MetaMaskInpageProvider),
+      }
+      await ArcxAnalyticsSdk.init('', {
+        ...ALL_FALSE_CONFIG,
+        trackTransactions: true,
+      })
+      expect(postRequestStub.getCall(0).calledWith(PROD_URL_BACKEND, '', '/identify')).to.be.true
+
+      await window.web3.currentProvider.request({
+        method: 'eth_sendTransaction',
+        params: transactionParams,
+      })
+      expect(
+        postRequestStub.getCall(1).calledWith(PROD_URL_BACKEND, '', '/submit-event', {
```

Comment:
Can use the helper function from the bottom `getAnalyticsData`

Reply:
I didn't notice